src/services/database/models/user.js

```diff
@@ -30,6 +30,13 @@ const UserSchema = new Schema(
       required: true,
       enum: ['male', 'female'],
     },
+    // Including gender as a field in the user model might be unnecessary:
```

Comment:
If this is your argument, would you consider removing phone number like we removed address? Phone number could be used as a login field but it's not necessary

Reply:
I would say only keep it if we are gonna use it for login or 2FA. I think we remove address and just keep zip code. Only need fields that would be relevant metadata